docs/getting_started/using_modin/using_modin_cluster.rst

```diff
@@ -108,4 +123,10 @@ you can customize your Ray environment for use in Modin!
 .. _`DataFrame`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html
 .. _`pandas`: https://pandas.pydata.org/pandas-docs/stable/
 .. _`open an issue`: https://github.com/modin-project/modin/issues
-.. _`Ray's documentation`: https://ray.readthedocs.io/en/latest/api.html
+.. _`Ray's API docs`: https://ray.readthedocs.io/en/latest/api.html
+.. _`Part 1`: https://github.com/modin-project/modin/tree/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/exercise_5.ipynb
+.. _`Part 2`: https://github.com/modin-project/modin/tree/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/exercise_6.ipynb
+.. _`Ray's autoscaler options`: https://docs.ray.io/en/latest/cluster/vms/references/ray-cluster-configuration.html#cluster-config
+.. _`Ray's cluster docs`: https://docs.ray.io/en/latest/cluster/getting-started.html
+.. _`NYC Taxi dataset`: https://modin-datasets.s3.amazonaws.com/testing/yellow_tripdata_2015-01.csv
+.. _`Modin's cluster setup config`: https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/modin-cluster.yaml
```

Comment:
It seems this file is stale and we should update it as part of a separate issue. Actually, we would need update cluster modes for all distr backends - Ray, Dask, MPI.

Reply:
> It seems this file is stale and we should update it as part of a separate issue.

Indeed. Maybe @Retribution98 can look at it? Here one will need to create a cluster in AWS.

> Actually, we would to need update cluster modes for all distr backends - Ray, Dask, MPI.

Don't get it.

Reply:
> > It seems this file is stale and we should update it as part of a separate issue.
> 
> Indeed. Maybe @Retribution98 can look at it? Here one will need to create a cluster in AWS.

We will schedule this offline.

> > Actually, we would to need update cluster modes for all distr backends - Ray, Dask, MPI.
> 
> Don't get it.

I mean docs/examples regarding Modin on cluster.